src/app/workbench/sidebar/DetailsSidebar.tsx

```diff
@@ -34,7 +34,15 @@ export function DetailsSidebar({ workbench }: IDetailsSidebarProps) {
     let currString = '';
 
     ordino.workbenches[workbench.index - 1].selection.forEach((s) => {
-      currString += `${s}, `;
+      const concatStr = ', ';
+      const prevFormatting = ordino.workbenches[workbench.index - 1].formatting;
+      if (prevFormatting) {
+        const selectionDataRow = ordino.workbenches[workbench.index - 1].data[s][prevFormatting.titleColumn || prevFormatting.idColumn];
+        currString += selectionDataRow + concatStr;
+      } else {
+        currString += s + concatStr;
+      }
+      console.log(currString);
     });
```

Comment:
```ts
const prevWorkbench = ordino.workbenches[workbench.index - 1];
const prevFormatting = prevWorkbench.formatting;
const currString = prevWorkbench.selection.map((selectedId) => {
  return (prevFormatting) ? prevWorkbench.data[selectedId][prevFormatting.titleColumn || prevFormatting.idColumn] : selectedId
}).join(', ');
```

This should do the same as your code. Untested.

* Extract `concatStr` and `prevFormatting` from the `forEach` as it remains the same for each iteration
* Store long repetitive strings such as `ordino.workbenches[workbench.index - 1]` in variables to improve readability
* If you want to concatinate items of an array use `.join()`
* Remove `console.log()`